pages/zh/lb4/Application.md

```diff
@@ -0,0 +1,220 @@
+---
+lang: zh
+title: '应用（Application）'
+keywords: LoopBack 4.0, LoopBack 4
+sidebar: lb4_sidebar
```

Comment:
Let's change **all** these translated pages' sidebar to `sidebar: zh_lb4_sidebar`.

Reply:
Changed all "sidebar"